tests/integration/cloudformation/api/test_transformers.snapshot.json

```diff
@@ -0,0 +1,27 @@
+{
+  "tests/integration/cloudformation/api/test_transformers.py::test_duplicate_resources": {
+    "recorded-date": "17-02-2023, 14:24:10",
+    "recorded-content": {
+      "api-details": [
+        {
+          "id": "<id:1>",
+          "name": "<name:1>",
+          "createdDate": "datetime",
+          "version": "1.2.3",
+          "apiKeySource": "HEADER",
+          "endpointConfiguration": {
+            "types": [
+              "EDGE"
+            ]
+          },
+          "tags": {
+            "aws:cloudformation:logical-id": "RestApi",
+            "aws:cloudformation:stack-id": "arn:aws:cloudformation:<region>:111111111111:stack/stack-8a6438b2/51efa190-aec6-11ed-a8ce-0a8057042631",
+            "aws:cloudformation:stack-name": "stack-8a6438b2"
```

Comment:
These don't seem stable. I.e. when running this against AWS the validation will fail. 
The stack name & uuid would need to be handled via transformers here to avoid this. :thinking:

Reply:
Sounds good. I've added `@pytest.mark.skip_snapshot_verify(paths=["$..tags"])` to avoid verification of the tags, but sounds like a good idea to move it to a transformer 👍 (will update..)

Reply:
skip_snapshot_verify does (intentionally) not work when verifying against AWS. It should generally only be used for skipping verification between AWS & LocalStack

Reply:
Gotcha, now I see what you're saying. Updated the transformers in the test. 👍